webgl-renderers/WebGLRenderer.js

```diff
@@ -777,14 +912,19 @@ WebGLRenderer.prototype.drawBuffers = function drawBuffers(vertexBuffers, mode,
  */
 WebGLRenderer.prototype.updateSize = function updateSize(size) {
     if (size) {
```

Comment:
When the mesh uniforms are updated (custom materials), the size method gets called and it doesn't need the size (refer to the handleMaterialInput function).